cantools/database/can/formats/dbc.py

```diff
@@ -556,7 +560,7 @@ def _dump_senders(database):
     return bo_tx_bu
 
 
-def _dump_comments(database):
+def _dump_comments(database, reverse_signals):
```

Comment:
as above: passing a `sort_fn` (or similar) function would be more general...

Reply:
I have implemented that in commit 3f0916797c6e7fdb0199a18b24e2d191d5513775. Unfortunately I'm not very experienced with typing yet and mypy isn't happy with what I've done. Do you know how to fix those errors?

Reply:
I have found the solution, mypy is happy now